resolwe_bio/processes/workflows/wgbs.yml

```diff
@@ -85,6 +85,17 @@
             Merging CpG pairs results in symmetric methylation levels. Methylation is usually
             symmetric (cytosines at CpG sites were methylated on both DNA strands). Choosing this
             option will only keep the CpG sites data.
+    - name: markduplicates
```

Comment:
Bump the corresponding process versions. This is important, since one could checkout this exact commit.

Reply:
Sorry, I was not clear enough. Bump the workflow version for the change introduced in this commit. Version bump in process `markduplicates` is not required.